content/articles/operators-In-c#/index.md

```diff
@@ -0,0 +1,400 @@
+
+The program below shows all the relational operators in use:
+
+```c#
+using System;
+namespace Relational_operators
+{
+
+    class examples
+    {
+        static void Main(string[] args)
+        {
+            bool outcome;
+            int a = 6, b = 12;
+            outcome = (a == b);
+            Console.WriteLine("Equal to Operator: " + outcome);
+            outcome = (a > b);
+            Console.WriteLine("Greater than Operator: " + outcome);
+            outcome = (a < b);
+            Console.WriteLine("Less than Operator: " + outcome);
+            outcome = (a >= b);
+            Console.WriteLine("Greater than or Equal to: " + outcome);
+            outcome = (a <= b);
+            Console.WriteLine("Lesser than or Equal to: " + outcome);
+            outcome = (a != b);
+            Console.WriteLine("Not Equal to Operator: " + outcome);
+        }
+    }
+}
+```
+
+This is the output:
+
+```bash
+Equal to Operator: False
+Greater than Operator: False
+Less than Operator: True
+Greater than or Equal to: False
+Lesser than or Equal to: True
+Not Equal to Operator: True
+```
+
+### Conditional operators
+A ternary operator is a simplified form of the if-else expression. The term ternary comes from the fact that it contains three operands.Contingent upon the value of a Boolean articulation, it will return one of two outcomes.
+
+The conditional operator code:
+
+```
+ condition ? first_expression : second_expression; 
+```
+
+The syntax explanation:
+
+**Condition**- It must be tested to see if it is true or untrue.
+
+**First expression**- If the boolean expression results to true, it is assessed and the result is obtained.
+
+**Second expression**- If the boolean expression results to false, it is assessed and the outcome is determined.
+
+The program below shows all the conditional operators in use:
+
+```c#
+using System;
+namespace Conditional_operators
+{
+
+    class example
+    {
+        static void Main(string[] args)
+        {
+            int a=6, b=7, outcome;
+            outcome = a>b ? a:b;
+            Console.WriteLine("final outcome: " + outcome);
+            outcome = a < b ? a : b;
+            Console.WriteLine("final outcome: " + outcome);
+        }
+    }
+}
+```
+
+This is the output:
+
+```bash
+final outcome: 7
+final outcome: 6
+```
+
+### Logical operators
+They're used to integrate two or more conditions/constraints or to supplement the evaluation of the original condition. The following is a list of them.
+
+1. **logical OR** (`||`): When one (or both) of the requirements in question are met, the ‘||' operator returns true. Otherwise, false is returned. If one of an o rb is true, for example, x || y returns true (i.e. non-zero). Of course, when both x and y are true, it returns true.
```

Comment:
```suggestion
1. **logical OR** (`||`): When one (or both) of the requirements in question are met, the ‘||' operator returns true. Otherwise, false is returned. For example, x || y returns true (i.e. non-zero). Of course, when both x and y are true, it returns true.
```